pkg/common/telemetry/specific_metrics.go

```diff
@@ -0,0 +1,428 @@
+	return StartCall(m, RegistrationAPI, Entry, Fetch)
+}
+
+// RegistrationAPIFetchFedBundleCall return a call counter
+// for server's registration API -> Fetch Federated Bundle
+func RegistrationAPIFetchFedBundleCall(m Metrics) *CallCounter {
+	return StartCall(m, RegistrationAPI, FederatedBundle, Fetch)
+}
+
+// RegistrationAPIListEntriesCall return a call counter
+// for server's registration API -> List Entries
+func RegistrationAPIListEntriesCall(m Metrics) *CallCounter {
+	// TODO called technically by multiple APIs; add more labels?
+	return StartCall(m, RegistrationAPI, Entry, List)
+}
+
+// RegistrationAPIListFedBundlesCall return a call counter
+// for server's registration API -> List Federated Bundles
+func RegistrationAPIListFedBundlesCall(m Metrics) *CallCounter {
+	return StartCall(m, RegistrationAPI, FederatedBundle, List)
+}
+
+// RegistrationAPIUpdateEntryCall return a call counter
+// for server's registration API -> Update Entry
+func RegistrationAPIUpdateEntryCall(m Metrics) *CallCounter {
+	return StartCall(m, RegistrationAPI, Entry, Update)
+}
+
+// RegistrationAPIUpdateFedBundleCall return a call counter
+// for server's registration API -> Update Federated Bundle
+func RegistrationAPIUpdateFedBundleCall(m Metrics) *CallCounter {
+	return StartCall(m, RegistrationAPI, FederatedBundle, Update)
+}
+
+// RotateAgentSVIDCall return a call counter for AgentSVID
+// Rotation. Takes the agent's spiffe ID as a label.
+func RotateAgentSVIDCall(m Metrics, id string) *CallCounter {
+	cc := StartCall(m, AgentSVID, Rotate)
+	AddSPIFFEID(cc, id)
+	return cc
+}
+
+// RotateServerSVIDCall return a call counter for
+// Server SVID Rotation.
+func RotateServerSVIDCall(m Metrics) *CallCounter {
+	return StartCall(m, SVID, Rotate)
+}
+
+// ServerCAManagerPrepareJWTKeyCall return a call counter for
+// Server CA Manager preparing a JWT Key
+func ServerCAManagerPrepareJWTKeyCall(m Metrics) *CallCounter {
+	// TODO add CA?
+	return StartCall(m, Manager, JWTKey, Prepare)
+}
+
+// ServerCAManagerPrepareX509CACall return a call counter for
+// Server CA Manager preparing an X509 CA
+func ServerCAManagerPrepareX509CACall(m Metrics) *CallCounter {
+	// TODO add CA?
+	return StartCall(m, Manager, X509CA, Prepare)
+}
+
+// WorkloadAPIFetchJWTSVIDCall return a call counter
+// for agent's Workload API -> Fetch JWT SVID
+func WorkloadAPIFetchJWTSVIDCall(m Metrics) *CallCounter {
+	cc := StartCall(m, WorkloadAPI, FetchJWTSVID)
+	cc.AddLabel(SVIDType, JWT)
+	return cc
+}
+
+// WorkloadAPIFetchX509SVIDCall return a call counter
+// for agent's Workload API -> Fetch X509 SVID
+func WorkloadAPIFetchX509SVIDCall(m Metrics) *CallCounter {
+	cc := StartCall(m, WorkloadAPI, FetchX509SVID)
+	cc.AddLabel(SVIDType, X509)
+	return cc
+}
+
+// End Call Counters
+
+// Counters (literal increments, not call counters)
+
+// ActivateJWTKeyManagerCounter increment counter for activation
+// of JWT Key manager
+func ActivateJWTKeyManagerCounter(m Metrics) {
```

Comment:
should these of methods be prefixed with `Incr` to be more verby?

Reply:
Yes, `Incr` prefix would probably help. `Activate` and `Increment` are different actions. `ActivateJWTKeyManagerCounter` makes me think i'm initializing a counter to be used later, not changing an existing value.